layout/_partial/comments.pug

```diff
@@ -144,4 +144,15 @@ if theme.minivaline.enable == true
     })
 
 if theme.utterances.enable == true
-  script(src='https://utteranc.es/client.js', repo=theme.utterances.repo, issue-term=theme.utterances.identifier, theme=theme.utterances.theme, crossorigin='anonymous', async)  
\ No newline at end of file
+  script(src='https://utteranc.es/client.js', repo=theme.utterances.repo, issue-term=theme.utterances.identifier, theme=theme.utterances.theme, crossorigin='anonymous', async)  
+
+if theme.twikoo.enable == true
+  #tcomment
+  script(src='https://cdn.jsdelivr.net/npm/twikoo/dist/twikoo.all.min.js')
+  script.
+    twikoo.init({
+      envId: '#{theme.twikoo.envId}',
+      el: '#tcomment',
+      region: '#{theme.twikoo.region}',
+      path: '#{theme.twikoo.path}'
```

Comment:
It seems that `#{theme.twikoo.path}` cannot  correspond to any item in `_config.yml`.

Reply:
Added `twikoo.path` to `_config.yml` & `README.md`.